build.gradle

```diff
@@ -51,7 +51,7 @@ configurations {
 dependencies {
     compileOnly 'com.google.code.findbugs:annotations:3.0.1'
     compile 'org.hibernate:hibernate-java8:5.0.12.Final'
-    compile 'org.opentestsystem.ap:ap-common:0.3.26'
+    compile 'org.opentestsystem.ap:ap-common:0.3.32-SNAPSHOT'
```

Comment:
Will update this when IAT does a release of their AP_Common library.  In the meantime, the organization fields are in the `develop` branch.